$MirOS: ports/print/cups/patches/patch-systemv_Makefile,v 1.1 2006/06/21 19:25:36 bsiegert Exp $
--- systemv/Makefile.orig	Wed Jul 11 23:46:19 2007
+++ systemv/Makefile	Wed Oct 22 00:06:35 2008
@@ -109,7 +109,7 @@ uninstall:
 
 accept:	accept.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o accept accept.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o accept accept.o $(LIBS)
 	$(RM) reject cupsenable cupsdisable
 	$(LN) accept reject
 	$(LN) accept cupsenable
@@ -122,7 +122,7 @@ accept:	accept.o ../cups/$(LIBCUPS)
 
 cancel:	cancel.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cancel cancel.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cancel cancel.o $(LIBS)
 
 
 #
@@ -131,7 +131,7 @@ cancel:	cancel.o ../cups/$(LIBCUPS)
 
 cupsaddsmb:	cupsaddsmb.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cupsaddsmb cupsaddsmb.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cupsaddsmb cupsaddsmb.o $(LIBS)
 
 
 #
@@ -140,7 +140,7 @@ cupsaddsmb:	cupsaddsmb.o ../cups/$(LIBCU
 
 cupsctl:	cupsctl.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cupsctl cupsctl.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cupsctl cupsctl.o $(LIBS)
 
 
 #
@@ -149,7 +149,7 @@ cupsctl:	cupsctl.o ../cups/$(LIBCUPS)
 
 cupstestdsc:	cupstestdsc.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ cupstestdsc.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ cupstestdsc.o $(LIBS)
 
 
 #
@@ -158,7 +158,7 @@ cupstestdsc:	cupstestdsc.o ../cups/$(LIB
 
 cupstestppd:	cupstestppd.o ../cups/$(LIBCUPS) ../filter/$(LIBCUPSIMAGE)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ cupstestppd.o $(LINKCUPSIMAGE) $(IMGLIBS) $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ cupstestppd.o $(LINKCUPSIMAGE) $(IMGLIBS) $(LIBS)
 
 
 #
@@ -167,7 +167,7 @@ cupstestppd:	cupstestppd.o ../cups/$(LIB
 
 lp:	lp.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lp lp.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lp lp.o $(LIBS)
 
 
 #
@@ -176,7 +176,7 @@ lp:	lp.o ../cups/$(LIBCUPS)
 
 lpadmin:	lpadmin.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lpadmin lpadmin.o $(LIBZ) $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lpadmin lpadmin.o $(LIBZ) $(LIBS)
 
 
 #
@@ -185,7 +185,7 @@ lpadmin:	lpadmin.o ../cups/$(LIBCUPS)
 
 lpinfo:	lpinfo.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lpinfo lpinfo.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lpinfo lpinfo.o $(LIBS)
 
 
 #
@@ -194,7 +194,7 @@ lpinfo:	lpinfo.o ../cups/$(LIBCUPS)
 
 lpmove:	lpmove.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lpmove lpmove.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lpmove lpmove.o $(LIBS)
 
 
 #
@@ -203,7 +203,7 @@ lpmove:	lpmove.o ../cups/$(LIBCUPS)
 
 lpoptions:	lpoptions.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lpoptions lpoptions.o $(LIBZ) $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lpoptions lpoptions.o $(LIBZ) $(LIBS)
 
 
 #
@@ -212,7 +212,7 @@ lpoptions:	lpoptions.o ../cups/$(LIBCUPS
 
 lppasswd:	lppasswd.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lppasswd lppasswd.o $(LIBZ) $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lppasswd lppasswd.o $(LIBZ) $(LIBS)
 
 
 #
@@ -221,7 +221,7 @@ lppasswd:	lppasswd.o ../cups/$(LIBCUPS)
 
 lpstat:	lpstat.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lpstat lpstat.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lpstat lpstat.o $(LIBS)
 
 
 #
